lib/internal/quic/core.js

```diff
@@ -2175,13 +2175,11 @@ class QuicSession extends EventEmitter {
     }, this);
 
     // Either the kReady event will be invoked or the 'error'
-    // event. Wait here for both.
-    const err = await Promise.race([
-      once(stream, kReady),
-      once(stream, 'error')
-    ]);
-    if (err && err.length > 0)
-      throw err[0];
+    // event. Wait here for either.
+    await (new Promise((resolve, reject) => {
+      stream.on(kReady, resolve);
+      stream.on('error', reject);
+    }));
```

Comment:
Not quite sure I understand this change here (in d554fefed48f2834c3c0aafbc432b43caa9e038c) – doesn’t `once` already automatically handle the `'error'` case, i.e. wouldn’t `await once(stream, kReady);` do the right thing?

Reply:
Ah right... forgot that `once()` handles the `'error'` event also here. good catch :-)